package.json

```diff
@@ -23,6 +23,7 @@
     "mixin-deep": "2.0.1",
     "moment": "2.30.1",
     "noty": "3.2.0-beta",
+    "oidc-client-ts": "^3.0.1",
```

Comment:
> For what it's worth, even with the range selector, an exact version would be pinned in `package-lock.json`.

What we've seen is that when depdendabot PRs regenerate `package-lock.json` it can pull in newer versions.